etl_manager/specs/glue_spark_dict.json

```diff
@@ -4,6 +4,7 @@
     "long" : {"glue" : "bigint", "spark": "LongType"},
     "float" : {"glue" : "float", "spark": "FloatType"},
     "double" : {"glue" : "double", "spark": "DoubleType"},
+    "decimal" : {"glue" : "decimal", "spark": "DecimalType"},    
```

Comment:
Style point – can this JSON be formatted?

Reply:
Resolved in commit [20ce794](https://github.com/moj-analytical-services/etl_manager/pull/132/commits/20ce794c3e9bdfd15e1e330a31f388b29974eba0).